red/Animal/lib/LivingCreature.pm

```diff
@@ -57,7 +56,7 @@ Returns a string representation of the sound the LivingCreature would make.
 
 =cut
 
-sub sound { die shift . '::sound has not been implemented' }
+requires 'sound';
```

Comment:
The ``requires`` tag is fairly useful for this sort of thing, although is trickier to fulfil with moosey attributes.